Submitted By: Tushar Teredesai <tushar@linuxfromscratch.org>
Date: 2004-01-24
Initial Package Version: 1.4.1
Origin: NA
Description: Link to libodbc. See unixodbc.org for downloading the lib.

--- Build-j2sdk-1.4.1-standard/j2se/make/sun/jdbc/Makefile.orig	2003-03-11 23:51:45.000000000 -0600
+++ Build-j2sdk-1.4.1-standard/j2se/make/sun/jdbc/Makefile	2003-03-11 23:59:27.000000000 -0600
@@ -29,10 +29,10 @@
 FILES_export = \
     sun/jdbc/odbc/JdbcOdbc.java 
 
-ifeq ($(PLATFORM),linux)
+#ifeq ($(PLATFORM),linux)
     # on linux you may not have an odbc library
-     INIT = make_libs
-endif
+     #INIT = make_libs
+#endif
 
 ifeq ($(PLATFORM),windows)
   # Files built here do not compile with warning level 3
